documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -650,6 +650,42 @@ XHCI_DEBUG=0x3
 
 Default: `0x0` (no USB debug messages enabled)
 
+[[SDRAM_BANKLOW]]
+==== `SDRAM_BANKLOW`
+
+SDRAM_BANKLOW controls how the SDRAM banks are arranged within the system address space. The bank bits may be mapped to fixed address bits in the mid part of the address, between row and column address bits, or in the MSBs of the address:
+This setting can significantly affect SDRAM performance.
+
+When set to 0 the bootloader will chose the preferred option. This is currently 3 on Pi 4 family and 1 on Pi 5 family.
+
+[cols="1m,3"]
+|===
+| BANKLOW | Address Bit Mapping
+
+| 0
+| Address = {bank[3:0], row, column}
```

Comment:
The text elsewhere says "When set to 0 the bootloader will choose the preferred option" - so why is there an explicit option for 0 listed in this table? :confused: